packages/foreman/rubygem-fog-ovirt/rubygem-fog-ovirt.spec

```diff
@@ -86,12 +89,17 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/CONTRIBUTING.md
 %doc %{gem_instdir}/CONTRIBUTORS.md
 %{gem_instdir}/Gemfile
+%{gem_instdir}/Gemfile.local.rb
```

Comment:
This one should be `%exclude`, it was added by mistake and will be removed in the next release.